basic-examples/bind-constants/subworkflow.yaml

```diff
@@ -0,0 +1,44 @@
+version: v0.2.0
+input:

```

Comment:
It might be worth having a block comment at the top here which basically says that the (sub)workflow accepts an input object with fields for an integer and an object, and it returns an object with those same three values at the same level (i.e., a bare-bones functional description).  I wouldn't say too much, because, aside from the fact that the code is pretty self-explanatory, we don't really want to draw too much attention away from `bindConstants()`.

Reply:
i'll add it